assets/config.js

```diff
@@ -4,36 +4,34 @@
 
 export default {
 	appTitle: "We Count",
-	appTitleShort: "WC",
-	appDescription: "We Count Project",
-	appThemeColor: "#fffffff",
-	appBgColor: "#00172c ",
-	appIcon: "assets/app-icon.png",
+	appShortTitle: "We Count",
+	appDescription: "Removing bias and exclusion in the data economy.",
+	appThemeColor: "#fffffF",
+	appBgColor: "#ffffff",
 	wpDomain: "https://wecount-cms.inclusivedesign.ca",
-	loadDbName: "starter_wp",
 	apiBase: "/wp-json/wp/v2/",
 	contactEmail: "wecount@inclusivedesign.ca",
 	// Using raw SVG content is to work around the issue with dynamically loading and injecting inline SVGs into the template.
 	socialMedias: [{
 		id: "facebook",
 		title: "Facebook",
 		href: "",
-		svg: "<svg width='78' height='78' viewBox='-2 -2 82 82' fill='none' xmlns='http://www.w3.org/2000/svg'><path d='M78 39C78 17.4586 60.5414 0 39 0C17.4586 0 0 17.4586 0 39C0 58.4695 14.2594 74.6027 32.9062 77.5277V50.2734H23.0039V39H32.9062V30.4078C32.9062 20.635 38.7258 15.2344 47.6379 15.2344C51.9035 15.2344 56.3672 15.9961 56.3672 15.9961V25.5938H51.4465C46.602 25.5938 45.0938 28.6025 45.0938 31.6875V39H55.9102L54.1811 50.2734H45.0938V77.5277C63.7406 74.6027 78 58.4695 78 39Z' fill='white'/></svg>"
+		svg: "<svg width='78' height='78' viewBox='-2 -2 82 82' xmlns='http://www.w3.org/2000/svg'><path d='M78 39C78 17.4586 60.5414 0 39 0C17.4586 0 0 17.4586 0 39C0 58.4695 14.2594 74.6027 32.9062 77.5277V50.2734H23.0039V39H32.9062V30.4078C32.9062 20.635 38.7258 15.2344 47.6379 15.2344C51.9035 15.2344 56.3672 15.9961 56.3672 15.9961V25.5938H51.4465C46.602 25.5938 45.0938 28.6025 45.0938 31.6875V39H55.9102L54.1811 50.2734H45.0938V77.5277C63.7406 74.6027 78 58.4695 78 39Z' fill='white'/></svg>"
```

Comment:
Just double checking that the intention here is to only remove `fill='none'` from `<svg>` and to leave the `fill='white'` on the `<path>`.

Reply:
Oops, missed that. Good catch. Fixed in a54afed.